compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -489,8 +489,38 @@ class OMR_EXTENSIBLE CodeGenerator
    bool isRegisterClobberable(TR::Register *reg, uint16_t count);
 
    // ilgen
-   bool ilOpCodeIsSupported(TR::ILOpCodes); // no virt, default, cast
 
+   /**
+    * @brief Returns if an IL OpCode is supported by current CodeGen
+    *
+    * @param op The IL OpCode being checked.
+    *
+    * @return True if the IL OpCode is supported otherwise false.
+    */
+   static bool ilOpCodeIsSupported(TR::ILOpCodes op);
```

Comment:
My main nit is with the naming of these functions.  Since this is a question of the code generator, wouldn't `isILOpCodeSupported` be more appropriate?  The current name is more of a statement than a question.

Reply:
This is an existing query and its name was not changed. Luckily, no known downstream projects use this query so that we can rename it. I've updated its name to `isILOpCodeSupported`.